### PR TITLE
Replaced fiberMag with fiberFlux

### PR DESCRIPTION
**Pull request by hassanxp**

### CODE REVIEW

File: python/pfs/drp/stella/fitReference.py [hunk 1]
```diff
@@ -185,26 +185,26 @@ def run(self, spectrum):
         spectrum : `pfs.datamodel.PfsReference`
             Reference spectrum.
         """
-        mag = set(spectrum.target.fiberMags.values())
-        if len(mag) != 1:
+        fiberFlux = set(spectrum.target.fiberFlux.values())
+        if len(fiberFlux) != 1:
             raise RuntimeError("This implementation requires a single ABmag, but was provided: %s" %
```
Comment (PaulPrice) [inline python/pfs/drp/stella/fitReference.py:190]: Fix message.

File: python/pfs/drp/stella/fitReference.py [hunk 2]
```diff
@@ -185,26 +185,26 @@ def run(self, spectrum):
...
                 break
         else:
-            raise RuntimeError(f"Unable to find bandpass for any of {spectrum.target.fiberMags.keys()}")
+            raise RuntimeError(f"Unable to find bandpass for any of {spectrum.target.fiberFlux.keys()}")
 
         ref = readAmbre(spectrum.target, spectrum.wavelength)
         # For now, only use the filter curve in calculating the expected flux.
         # More precise work in the future may fold in the CCD QE, the mirror and the lens barrel.
-        ref *= 10**(-0.4*(mag + 48.6))*bandpass.integrate()/bandpass.integrate(ref)  # erg/cm^2/s/Hz
+        ref *= 10**(-0.4*(fiberFlux + 48.6))*bandpass.integrate()/bandpass.integrate(ref)  # erg/cm^2/s/Hz
```
Comment (PaulPrice) [inline python/pfs/drp/stella/fitReference.py]: Fix calculation.

File: python/pfs/drp/stella/synthetic.py [hunk 3]
```diff
@@ -290,12 +291,13 @@ def makeSyntheticPfsConfig(config, pfsDesignId, visit, rng=None,
 
     fiberStatus = np.full_like(targetType, FiberStatus.GOOD)
 
-    fiberMag = [np.array([22.0, 23.5, 25.0, 26.0] if
-                         tt in (TargetType.SCIENCE, TargetType.FLUXSTD) else [])
-                for tt in targetType]
+    fbr_flx = [(m*u.ABmag).to_value(u.nJy) for m in [22.0, 23.5, 25.0, 26.0]]
```
Comment (PaulPrice) [inline python/pfs/drp/stella/synthetic.py:294]: Fix `fbr_flx`.

**Review by PaulPrice — APPROVED** [review]

File: python/pfs/drp/stella/synthetic.py (hunk 3 above)
Comment (hassanxp) [reply python/pfs/drp/stella/synthetic.py:296]: This calculation computes the flux in [nJy] from the AB magnitude. Can you elaborate a little more as to the error?

**Pull request merged**